app/views/planning_applications/consultee/responses/_consultee_tabs.html.erb

```diff
@@ -0,0 +1,34 @@
+<% show_action = local_assigns.fetch(:show_action, true) %>
```

Comment:
[Strict locals](https://guides.rubyonrails.org/action_view_overview.html#strict-locals) might work nicely here, and in the other partial:

```suggestion
<%# locals(show_action: true) %>
```

Reply:
I don't want to strict local to true, as it depends on the view